ReSharper.FSharp/test/data/features/daemon/interpolatedStringCandidate/Not available 07 - Other string.fs.gold

```diff
@@ -0,0 +1,7 @@
+﻿module Module
+
+let foo = id "foo"
+let s = sprintf |"%d %s"|(0) 5 foo
+
+---------------------------------------------------------
+(0): ReSharper Suggestion: Format string can be replaced with an interpolated string
```

Comment:
I wonder why it's called Not available? 🙂

Reply:
Think it was for the `let foo = ...` line above - although in hindsight it's a bit of a pointless test.